tests/sles4sap/hana_test.pm

```diff
@@ -46,6 +46,12 @@ sub run {
 
     assert_script_run "HDB info";
 
+    # Check for bsc#1031355
+    # UserTasksMax should be set to "infinity" in /etc/systemd/logind.conf.d/sap.conf
+    my $uid = script_output = "id -u";
+    my $pids_max = script_output "cat /sys/fs/cgroup/pids/user.slice/user-${uid}.slice/pids.max";
+    die "UserTasksMax != infinity" unless ($pids_max == "max");
```

Comment:
Be careful because `script_output` can be set to undefined (but should be ok here I think).

Reply:
Can be set to `undef` or can contain extra characters. Are you sure `$pids_max == "max"` will be enough?

Reply:
@alvarocarvajald Yes.